app/controllers/api_controller.rb

```diff
@@ -181,4 +181,20 @@ def update  # POST, PUT, PATCH
   def destroy # DELETE
     redirect_api_request(:destroy)
   end
+
+  def create_bundle_resource_blueprints(type, id, data)
```

Comment:
@bzwei if you can ignore the parsing part of this method, this is how I imagined the flow for creating a bundle. Can you let me know if the actual implementation will differ?